scheduling_resources/scheduling-logging-resources.adoc

```diff
@@ -0,0 +1,44 @@
+include::_attributes/common-attributes.adoc[]
+
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-06-24
+:_mod-docs-content-type: ASSEMBLY
+
+[id="scheduling-logging-resources"]
+= Scheduling logging resources
+
+toc::[]
+
+:context: scheduling-logging-resources
+
+You can control logging resources scheduling by defining node selectors, taints and tolerations, and affinity and anti-affinity configurations. 
+
+Node selector:: A node selector specifies a map of key/value pairs that are defined using custom labels on nodes and selectors specified in pods. For the pod to be eligible to run on a node, the pod must have the same key/value node selector as the label on the node.
+
+Taints and toleration:: Taints and tolerations allow the node to control which pods should, or should not, be scheduled on them.
+
+Affinity and anti-affinity:: Pod affinity and pod anti-affinity allow you to constrain which nodes your pod is eligible to be scheduled on based on the key-value labels on other pods.
+
+[IMPORTANT]
```

Comment:
It isn't technically wrong to callout this information in an IMPORTANT admonition. However, this isn't just a single consideration. With that in mind, I would move these into their own module or add them to the assembly with another heading `Considerations when using node selectors and node affinity rules`.

Reply:
removed the text about nodeSelectorTerms, so this is a single consideration now.